RELEASING.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Releasing
+route: /guides/releasing
+menu: Guides
+---
+
+# Releasing
+
+The release process is fully automated. The only condition is that contributors
+must follow the [contributing guidelines](/guides/contributing).
+
+How it works:
+
+- Pull requests are automatically labelled based on branch name. The labels are
+  used for:
+
+  - resolving next [semantic version number](https://semver.org)
+    (BREAKING.FEATURE.PATCH)
+  - grouping changes in changelog
+
+- Release changelog is generated as pull requests are merged into the `master`
+  branch. Invididual PR names are listed and grouped by type based on the label
+  added to them previously.
+
+See the source of `.github/workflows` for details.
+
+## Release Process
+
+**As you go:** make sure all contributions follow the contributing
+guidelines, especially the [Git workflow](/guides/contributing#git-workflow)
+(correct PR names and branch names). Check out the release draft on
+[GitHub releases page](https://github.com/react-ui-org/react-ui/releases) to see
+what the changelog looks like and what will be the next version number.
+
+1. **Manual:** once you are ready to publish a release, bump the version number
+   in `package.json` and `package-lock.json`. Make sure it matches the resolved
+   version number in the release draft. Commit as `Bump version` in
+   `release/<VERSION_NUMBER>` branch, create a pull request and merge it.
+
+2. **Automatic:** once the release pull request from step 1 is merged, the
+   following actions are triggered automatically:
+
+   1. GitHub release draft is published.
+   2. Git tag with version number is added to `master`.
+   3. Package is built and published to npm package registry.
+   4. Documentation is built and deployed to production.
```

Comment:
Unfortunately, this is the only step I know it works — see react-ui.io. The rest is blocked by an unknown error in Release Drafter workflow.